### PR TITLE
chore: add pipcompile command

### PR DESCRIPTION
**Pull request by garguelles**
# Changes

- add pipcompile command to update requirements.txt when installing new deps via docker

### CODE REVIEW

File: dev [hunk 1]
```diff
@@ -312,6 +313,11 @@ create_fake_bigdata() {
     docker compose run --rm misago python manage.py createfakehistory 2190 120
 }
 
+# Pip compile
```
Comment (rafalp) [inline dev:316]:
```suggestion
# Shortcut for pip compile new requirements.txt
```

**Review by rafalp — CHANGES_REQUESTED** [review]
LGTM but I have small nit

**Comment by garguelles** [comment]
Done. thanks!

File: dev [hunk 2]
```diff
@@ -312,6 +313,11 @@ create_fake_bigdata() {
     docker compose run --rm misago python manage.py createfakehistory 2190 120
 }
 
+# Shortcut for pip compile new requirements.txt
+pip_compile() {
+    docker compose run --rm misago pip-compile --output-file=requirements.txt requirements.in
+}
+
 # Command dispatcher
```
Comment (rafalp) [inline dev:321]: You also need to add dispatch for `pip_compile` here.

**Review by rafalp — CHANGES_REQUESTED** [review]

**Review by rafalp — APPROVED** [review]

**Comment by rafalp** [comment]
Thanks!

**Pull request merged**